mappings/net/minecraft/util/MobUtils.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_5493 net/minecraft/util/MobUtils
```

Comment:
Hmm, we usually only put really generic, i.e. not-that-much game related stuff in util. This thing we usually put in the entity package. In addition, I guess we might come up with a more specific name?

Reply:
Hmm... I guess I have to put it in `net.minecraft.entity.mob` package and rename it. Let me take a century 😄   
Plus. What about the `LoadingScreenOverlay`? I have just pushed it on.

Reply:
Hmm, I've named this class in #2030. I think my pr is more comprehensive as it covers the related code as a system.